ui/scss/component/_card.scss

```diff
@@ -183,26 +187,65 @@ $height-card-small: $spacing-vertical * 15;
 }
 
 .card-row {
+  + .card-row {
+    margin-top: $spacing-vertical * 1/3;
+  }
+}
+.card-row__items {
   > .card {
     vertical-align: top;
     display: inline-block;
     margin-right: $spacing-vertical / 3;
   }
-  + .card-row {
-    margin-top: $spacing-vertical * 1/3;
+  > .card:last-child {
+    margin-right: 0
   }
 }
 .card-row--small {
-  overflow-x: auto;
-  overflow-y: hidden;
+  overflow: hidden;
   white-space: nowrap;
-
+  position: relative;
   /*hacky way to give space for hover */
-  padding-left: 20px;
-  margin-left: -20px;
+  padding-right: 20px;
+  margin-right: -20px;
+}
+.card-row__items {
+  width: 100%;
+  overflow: hidden;
+  /*hacky way to give space for hover */
+  padding-top: 20px;
+  margin-top: -20px;
   padding-right: 20px;
   margin-right: -20px;
 }
 .card-row__header {
   margin-bottom: $spacing-vertical / 3;
+}
+
+.card-row__nav {
+  display: none;
+  position: absolute;
+  padding: 0 $spacing-vertical / 1.5;
+  top: ($font-size * 1.4 * $font-line-height) + ($spacing-vertical / 3);
+  height: ($width-card-small * 9 / 16) + ($font-size * 0.82 * $font-line-height * 4) + ($spacing-vertical * 4/3)
+}
+.card-row__nav .icon {
```

Comment:
A hover effect on this would also be nice. Maybe slightly larger size and icon size, as well as changing the icon color?